CONTRIBUTION.md

```diff
@@ -0,0 +1,235 @@
+# Contribution Guidelines for Ping SDK for Android
+
+Welcome! We are excited that you are interested in contributing to the **Ping SDK for Android**. 
+This document will guide you through the steps required to contribute to the project. Your 
+contribution is very valuable for us.
```

Comment:
I think we can delete this last sentence. Our appreciation is covered by the following sentence.